terralab/commands/pipelines_commands.py

```diff
@@ -46,35 +51,62 @@ def details(pipeline_name: str, version: int) -> None:
     """Get information about the PIPELINE_NAME pipeline"""
     pipeline_info = pipelines_logic.get_pipeline_info(pipeline_name, version)
 
-    # format the information nicely
-    col_width = 20
-
-    LOGGER.info(
-        f"{pad_column("Pipeline Name:", col_width)}{pipeline_info.pipeline_name}"
-    )
-    LOGGER.info(
-        f"{pad_column("Pipeline Version:", col_width)}{pipeline_info.pipeline_version}"
-    )
-    LOGGER.info(f"{pad_column("Description:", col_width)}{pipeline_info.description}")
-    LOGGER.info(
-        f"{pad_column("Min Quota Consumed:", col_width)}{pipeline_info.pipeline_quota.min_quota_consumed} {pipeline_info.pipeline_quota.quota_units}"
-    )
-    LOGGER.info("Inputs:")
-
-    inputs_for_usage = []
-    for input_definition in pipeline_info.inputs:
-        LOGGER.info(
-            f"{pad_column("", col_width)}{input_definition.name} ({input_definition.type})"
-        )
-        inputs_for_usage.extend([f"--{input_definition.name}", "YOUR_VALUE_HERE"])
-    inputs_string_for_usage = " ".join(inputs_for_usage)
-
-    LOGGER.info("Outputs:")
-    for output_definition in pipeline_info.outputs:
+    if pipeline_info:
```

Comment:
if there isnt pipeline info what happens?  Is this actually something we expect to be possible?

Reply:
no it's not... I assume this was added because of typing requirements but I think it's not necessary now. good catch. if the pipeline isn't found then this will error out above, so if we get here then we should have pipeline_info.